src/Microsoft.AspNetCore.Server.Kestrel/Infrastructure/Constants.cs

```diff
@@ -28,32 +28,36 @@ internal class Constants
 
         private static int? GetECONNRESET()
         {
-            switch (PlatformServices.Default.Runtime.OperatingSystemPlatform)
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                case Platform.Windows:
-                    return -4077;
-                case Platform.Linux:
-                    return -104;
-                case Platform.Darwin:
-                    return -54;
-                default:
-                    return null;
+               return -4077;
             }
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
```

Comment:
It's clearer that exactly one condition will match.  It's logically the same if every condition has a `return` statement, but it's still clearer to use us `if / else if / else` rather than `if / if / if`.